src/js/component.js

```diff
@@ -730,22 +731,30 @@ class Component {
    * @return {Component}
    */
   triggerReady() {
+
+    // // Ready should only be fired once
```

Comment:
@dmlap, I had this change in there but then decided to hold on it until we figured out how we wanted to handle tech changes. I forget where we landed, but also realized that the readyQueue does reset itself after ready is triggered, so ready listeners won't fire again on a tech change. That is, unless you listen for the ready event, not pass the function to ready(fn).

But basically the ready event was meant to let you add new ready listeners when the tech changes, it just may not be so clear that's the purpose. Maybe a `techloaded` event is the answer.